monitoring/boot_config_linux.go

```diff
@@ -230,9 +238,10 @@ type kernelVersionReader func() (version string, err error)
 type bootConfigReader func(release string) (io.ReadCloser, error)
 
 func parseKernelVersion(input string) (*KernelVersion, error) {
```

Comment:
Could you add a comment on this function explaining what format of kernel version string it expects?